.github/workflows/update-docs.yaml

```diff
@@ -7,10 +7,12 @@ jobs:
     name: Upload to documentation
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@dc323e67f16fb5f7663d20ff7941f27f5809e9b6
+      # ratchet:actions/checkout@v3.5.2
+      - uses: actions/checkout@8e5e7e5ab8b370d6c329ec480221332ada57f0ab
         name: checkout repo content
       - name: setup python
-        uses: actions/setup-python@75f3110429a8c05be0e1bf360334e4cced2b63fa
+        # ratchet:actions/setup-python@v4.6.0
```

Comment:
@yuleib please, leave the comment (`# ratchet:..`) in the same line of the action name.
It is used by the tool to updated the pinned version during the time.